testing.rb

```diff
@@ -0,0 +1,389 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/BlockNesting,Metrics/MethodLength,Metrics/ModuleLength,Metrics/CyclomaticComplexity,Metrics/PerceivedComplexity,Metrics/AbcSize
+
+# module Enumerable
+module Enumerable
+  def my_each
+    return to_enum(:my_each) unless block_given?
+
+    arr = to_a if self.class == Range
+    arr = self if self.class == Array
+    arr = self if self.class == Hash
+    n = 0
+    while n < arr.length
+      yield(arr[n])
+      n += 1
+    end
+    self
+  end
```

Comment:
- #my_each when a block is given when self is a hash behaves like #each calls the given block once for each element in self

expected: "[:a, 1][:b, 2][:c, 3][:d, 4][:e, 5]"
            got: ""

Reply:
I don't understand your suggestion at all. The project says use multiply_els only for the inject method, not for every method, so how are we going to run the code if we comment lines 7-19